### ISSUE TITLE
Case mismatch leads to false negatives

### ISSUE THREAD

**Issue by lorin** [dcramer/piplint#1]
If the case in a requirements file doesn't match the case in the "pip freeze" output, then piplint will incorrectly report an error. For example, if you have a requirements.txt file that looks like:

```
pygments
piplint
```

And then you do:

```
pip install -r requirements.txt
piplint requirements.txt
```

You'll get an error:

```
Requirement 'pygments' not found in virtualenv.
```
